### PR TITLE
Rebased:Add a way to connect 2 clients on one PC

### PR DESCRIPTION
**Pull request by sum2012**
This is not for Merging as #7276 
I tested Metal_Slug_XX only work in menu

### CODE REVIEW

**Comment by Bigpet** [comment]
Isn't this a little chaotic to open another pull request for this?

**Comment by sum2012** [comment]
I want to keep not conflict

**Comment by Bigpet** [comment]
Well the git way to do this would have been to make a pull request on  my branch, so it would stay in one github issue, but whatever. What's done is done.

File: UI/GameSettingsScreen.cpp [hunk 1]
```diff
@@ -392,10 +392,13 @@ void GameSettingsScreen::CreateViews() {
 	networkingSettings->Add(new ItemHeader(ms->T("Networking")));
 
 	networkingSettings->Add(new CheckBox(&g_Config.bEnableWlan, n->T("Enable networking", "Enable networking/wlan (beta)")));
+	networkingSettings->Add(new CheckBox(&g_Config.bBindLocal, n->T("Enable bind", "Specify client address")));
 
 #ifdef _WIN32
+	networkingSettings->Add(new PopupTextInputChoice(&g_Config.localBindAddress, n->T("Change bind address"), "", 255, screenManager()));
 	networkingSettings->Add(new PopupTextInputChoice(&g_Config.proAdhocServer, n->T("Change proAdhocServer Address"), "", 255, screenManager()));
 #else
+	networkingSettings->Add(new ChoiceWithValueDisplay(&g_Config.localBindAddress, n->T("Change bind address"), nullptr))->OnClick.Handle(this, 255, &GameSettingsScreen::OnChangeBindAddress);
```
Comment (Bigpet) [inline UI/GameSettingsScreen.cpp]: You're probably gonna need to remove the ",255" here

Comment (sum2012) [inline UI/GameSettingsScreen.cpp]: OK

**Comment by Bigpet** [comment]
Well Linux compiles but symbian and Android don't like the C++11 `std::to_string`

**Comment by sum2012** [comment]
Test Virtua Tennis 3 ,work

**Comment by sum2012** [comment]
@Bigpet I have fixed Android build

**Comment by lightfirst** [comment]
When I try leave lobby, I get this error: 

E[NET]: HLE\proAdhoc.cpp:1484 Socket error (10048) when connecting to 127.0.0.1/127.0.0.1:27312

connecting loss and cant reconnect unless you change pro AdhocServer IP address (et. 127.0.0.1 to 127.0.0.255)

**Comment by unknownbrackets** [comment]
Probably should be using SO_REUSEADDR or something.

-[Unknown]

**Comment by GAMELASTER** [comment]
I tested this and its works good on GTA Liberty City Stories!

**Comment by DonelBueno** [comment]
Why was this closed and deleted?

**Comment by sum2012** [comment]
Omg,I delete wrong branch in github software,will try to recover

**Comment by hrydgard** [comment]
try reflog!

**Comment by Bigpet** [comment]
the git hash is right there, just do "git checkout 37f12ca8a22c1618444a00b256b722e82ebb1065" and then "git checkout -b new_branchname"

**Comment by sum2012** [comment]
Very thanks @Bigpet

**Pull request closed without merging**